modules/jwplayerRtdProvider.md

```diff
@@ -122,7 +122,7 @@ where:
         - `ext.segtax` whose `502` value is the unique identifier for JW Player's proprietary taxonomy
         - `ext.cids` is an array containing the list of extended content ids as defined in [oRTB's community extensions](https://github.com/InteractiveAdvertisingBureau/openrtb/blob/master/extensions/community_extensions/extended-content-ids.md#example---content-id-and-seller-defined-context). 
         - `segment` is an array containing the segment taxonomy values as an object where:
-          - `id` is the string representation of the data segment value. 
+          - `value` is the string representation of the data segment. 
```

Comment:
id is correct according to the spec.

Reply:
I can't tell which is better from the spec:
id: ID of the data segment specific to the data provider.
value: String representation of the data segment value.

Is `id` more commonly used ?

Reply:
id is in the spec, value is not https://github.com/InteractiveAdvertisingBureau/openrtb/blob/master/extensions/community_extensions/segtax.md